mii/client.py

```diff
@@ -156,6 +166,48 @@ def destroy_session(self, session_id):
             client.destroy_session(session_id)
 
 
+class MIINonPersistentClient():
+    def __init__(self, task, deployment_name):
+        self.task = task
+        self.deployment_name = deployment_name
+
+    def query(self, request_dict, **query_kwargs):
+        task_methods = GRPC_METHOD_TABLE[self.task]
+        inference_pipeline = mii.non_persistent_models[self.deployment_name][0]
+
+        if self.task == Tasks.QUESTION_ANSWERING:
+            if 'question' not in request_dict or 'context' not in request_dict:
+                return
+
+            return task_methods.run_inference(inference_pipeline,
+                                              request_dict,
+                                              query_kwargs,
+                                              True)
```

Comment:
What is this `True`? This should cause an error since `run_inference` only takes 3 arguments.

Reply:
There is an error for some of the non_persistent tests when I try and pass *args, it will take the string that is passed in, and treat every character in the string as an argument and throw an error.